backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacGroup.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.cloud.notifications.auth.rbac;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.resteasy.reactive.ClientWebApplicationException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+import java.time.Duration;
+import java.util.UUID;
+
+@ApplicationScoped
+public class RbacGroup {
+
+    @Inject
+    @RestClient
+    RbacServer rbacServer;
+
+    @ConfigProperty(name = "rbac.enabled", defaultValue = "true")
+    Boolean isRbacEnabled;
```

Comment:
Nitpick: could be a primitive.
```suggestion
    boolean isRbacEnabled;
```